docs/source/develop.rst

```diff
@@ -91,7 +91,8 @@ Install dask and dependencies::
 
 For development dask uses the following additional dependencies::
 
-   pip install pytest moto mock
+   pip install pytest moto mock partd sparse
+   conda install cloudpickle
```

Comment:
ok, my bad - `zsh` with `oh-my-zsh` did some magic to `pip install -e .[complete]` which meant that the dependencies were not installed.